dashboard/modules/resource/resource_utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, List, Tuple
+
+ResourceSet = Dict[str, float]
+ResourceCountList = List[Tuple[ResourceSet, int]]
+
+
+def add_resource_usage_to_task(
+    task: str,
+    resource_set: ResourceSet,
+    task_resource_count_mapping: Dict[str, ResourceCountList],
+):
+    if task in task_resource_count_mapping:
+        task_resource_count_mapping[task] = _add_to_resources_list(
+            resource_set,
+            task_resource_count_mapping[task],
+        )
+    else:
+        task_resource_count_mapping[task] = [[resource_set, 1]]
+
+
+def _add_to_resources_list(
```

Comment:
It seems like we have a duplicated method . Why don't we just delete this file? (I think it makes more sense to have these methods within state folder than dashboard)